src/components/Breadcrumbs/BreadcrumbSeparator.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <li class="sb-breadcrumbs__separator" :class="computedClass">
+    <SbIcon size="normal" name="chevron-right" />
+  </li>
+</template>
+
+<script>
+import SbIcon from '../Icon/SbIcon.vue'
+export default {
+  name: 'BreadcrumbSeperator',
+  components: { SbIcon },
+  props: {
+    active: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  computed: {
+    computedClass() {
+      return {
+        'sb-breadcrumbs__separator--active': this.active,
+      }
+    },
+  },
+}
+</script>
+
+<style lang="scss">
+.sb-breadcrumbs__separator {
```

Comment:
May we add an external file for those?
It seems better to keep consistency, no matter how many lines we have

What do you think?